packages/frontend/core/src/modules/explorer/views/nodes/folder/index.tsx

```diff
@@ -567,6 +567,15 @@ export const ExplorerFolderNodeFolder = ({
     [handleDeleteChildren, t]
   );
 
+  const handleCollapsedChange = useCallback((collapsed: boolean) => {
+    if (collapsed) {
+      setNewFolderId(null); // reset new folder id to clear the renaming state
```

Comment:
I did not check the impl but maybe this can be resetted via blur event?

Reply:
Event on the input are handled in a deeper node, and I think it's not worth to adding another callback just for this scene.